roles/openshift_node/tasks/install.yml

```diff
@@ -1,4 +1,42 @@
 ---
+- name: Retrieve cluster machineconfigs
+  command: >
+    oc get machineconfigs
+    --kubeconfig={{ openshift_node_kubeconfig_path }}
+    --output=name
+  delegate_to: localhost
+  run_once: true
+  register: oc_get_machineconfigs
+  until:
+  - oc_get_machineconfigs.stdout != ''
+  changed_when: false
+
+- name: Set fact rendered_worker_machineconfig
+  set_fact:
+    rendered_worker_machineconfig: "{{ oc_get_machineconfigs.stdout | regex_search('^.*rendered-worker.*$', multiline=True)}}"
+  run_once: true
```

Comment:
we can actually find our rendered machine config by using 

```
$ oc get machineconfigpool worker --output=jsonpath='{.status.configuration.name}'
```

see the printer column for the machineconfigpool https://github.com/openshift/machine-config-operator/blob/4fe8559913b80253827df205b9e389ea309fc39c/install/0000_80_machine-config-operator_01_machineconfigpool.crd.yaml#L10-L12

Reply:
Updated.